TypeScript.YAML-tmLanguage

```diff
@@ -1624,11 +1624,12 @@ repository:
     - match: '{{startOfIdentifier}}(as)\s+(const)(?=\s*($|[;,:})\]]))'
       captures:
         '1': { name: keyword.control.as.ts }
-        '2': { name: storage.modifier.ts }
+        '2': { name: storage.modifier.const.type.ts }
     - begin: '{{startOfIdentifier}}(as)\s+'
       beginCaptures:
         '1': { name: keyword.control.as.ts }
       end: (?=$|^|[;,:})\]]|\|\||\&\&|({{startOfIdentifier}}(as)\s+)|(\s+\<))
+      contentName: meta.type.cast.ts
```

Comment:
Name this as `cast.expr.ts` and use `name` instead of `contentName` since cast expression starts at `as`